packages/apps/docs/src/docs/deploy/chainweb/chainweb-get-started.md

```diff
@@ -22,13 +25,487 @@ Before installing Chainweb, you should verify that your computer meets the follo
+
+You can download compressed archive files with `chainweb-node` release binaries for Ubuntu Linux directly from the [Releases](https://github.com/kadena-io/chainweb-node/releases) pages in the [chainweb-node](https://github.com/kadena-io/chainweb-node/) repository.
+If you have Ubuntu 20.0.4 or Ubuntu 22.04 on a physical or virtual machine, downloading the binary is the most straightforward installation path.
+
+To install from a release archive:
+
+1. Open a terminal shell on the physical or virtual host with the Ubuntu Linux operating system.
+
+2. Update the system with the latest software by running the following command:
+   
+   ```bash
+   sudo apt update && sudo apt upgrade
+   ```
+
+3. Install the required packages by running the following command:
+   
+   ```bash
+   sudo apt-get install ca-certificates libgmp10 libssl3 libsnappy1v5 zlib1g liblz4-1 libbz2-1.0 libgflags2.2 zstd
+   ```
+
+   Note that Ubuntu 20.04 requires the `libssl1.1` package instead of the `libssl3` package. 
+   This package has been deprecated in Ubuntu 22.04.
+   If your computer uses the Ubuntu 20.04 operating system, install the `libssl1.1` package instead of the `libssl3` package. 
+
+4. Download the archive file from the [Releases](https://github.com/kadena-io/chainweb-node/releases) page.
+   
+   Note that the archive file naming convention includes the `chainweb-node` version, compiler version, Ubuntu version, and a commit hash identifier using the following format:
+   
+   ```text
+   chainweb-<version>.ghc-<version>.ubuntu-<version>.<revision>.tar.gz
+   ```
+   
+   For example, the `chainweb-node` archive file for Ubuntu 22.04 looks like this:
+   
+   ```text
+   chainweb-2.24.1.ghc-9.6.5.ubuntu-22.04.89b0ac3.tar.gz
+   ```
+
+5. Change to the directory that contains the downloaded file—typically, the Downloads folder—or move the file to a new location.
+
+6. Unzip the compressed archive by running a command similar to the following:
+   
+   ```bash
+   gunzip chainweb-2.24.1.ghc-9.6.5.ubuntu-22.04.89b0ac3.tar.gz
+   ```
+
+7. Extract the archive by running a command similar to the following:
+   
+   ```bash
+   tar -xvf chainweb-2.24.1.ghc-9.6.5.ubuntu-22.04.89b0ac3.tar
+   ```
+
+8. Verify the binary is ready to use and review command-line configuration options by running the following command:
+   
+   ```bash
+   ./chainweb-node --help
+   ```
+
+   You should see usage information about the configuration settings you can specify as command-line options similar to the following truncated output:
+
+   ```bash
+   Usage: chainweb-node [--info] [--long-info] [-v|--version] [--license] 
+                        [-?|-h|--help] 
+                        [--print-config-as full|minimal|diff | --print-config] 
+                        [--config-file FILE] 
+   ```
+
+   From the usage information, you can see that there are a large number of configuration options that you can use to control and operation and behavior of the Chainweb node. 
```

Comment:
As an additional note, I usually recommend that people first run `./chainweb-node --print-config-as=full > config.yaml` or something, so they can have a full config to edit, without needing to run that command or grep through options every time

Reply:
Oh, I guess you do this later - nevermind

Reply:
I'm still on the bubble about this approach vs duplicating some of the information for each installation option separately.
Right now, it's organized like this:
- Get to the point where you have a binary and can view help (separate ways you can do that).
- Save and review the default config
- Edit the config
- Start the node with the config you want to use

But I'm not sure if it would be better to go with something like:

Install release binary
- Download and extract binary and view help.
- Save and review the default config
- Edit the config
- Start the node with the config you want to use

Start in a Docker container
- Download the Docker image
- View help
- Save and review the default config
- Edit the config
- Start the node with the config you want to use

Build from source with Haskell toolchain
- Download or update the Haskell toolchain
- Download and extract the source code
- Build the binary and view help.
- Save and review the default config
- Edit the config
- Start the node with the config you want to use

Build from source with Nix
- Download or update Nix
- Update the nix.conf
- Build the binary and view help.
- Save and review the default config
- Edit the config
- Start the node with the config you want to use